src/main/java/io/jenkins/plugins/signpath/SignPathPluginGlobalConfiguration.java

```diff
@@ -19,7 +19,7 @@
 public class SignPathPluginGlobalConfiguration extends GlobalConfiguration {
 
     private String apiURL = PluginConstants.DEFAULT_API_URL;
-    private String defaultTrustedBuildSystemCredentialId = PluginConstants.DEFAULT_TBS_CREDENTIAL_ID;
+    private String trustedBuildSystemCredentialId = PluginConstants.DEFAULT_TBS_CREDENTIAL_ID;
```

Comment:
Maybe rename the constant variable as well?

Reply:
The constant name is correct, it contains the default (initial) value for the global configuration property